connectomics/data/utils/data_segmentation.py

```diff
@@ -29,7 +29,7 @@ def relabel(seg, do_type=False):
     if do_type:
         m_type = getSegType(mid)
     mapping = np.zeros(mid, dtype=m_type)
-    mapping[uid] = np.arange(len(uid), dtype=m_type)
+    mapping[uid] = np.arange(1, len(uid) + 1, dtype=m_type)
```

Comment:
error case: uid has 0

Reply:
uid=uid[uid>0]

Reply:
I tried reverting this change in line 32, and instead added `uid=uid[uid>0]`, but the problem persists. So should I add `uid=uid[uid>0]` without reverting the line 32 change?

Reply:
the solution should be: add uid=uid[uid>0] and then your change (numbering things from 1)